### PR TITLE
Fix copy oc command for resources in debug dropdown

### LINKED ISSUE #1033: Enable 'copy oc get command' for resources other than plan/migrate
Inside the TreeActionsDropdown, we build an oc command for getting a specific resource from the oc cli. We need to decide where the  resource lives (source or destination) & indicate to the user which cluster to run the oc command on. 

Examples of resources we will be presented with:

```
kind: "Restore"
namespace: "openshift-migration"
name: "2d664340-1483-11eb-9b20-99dbdd85b490-mltbn"
children: (6) [{…}, {…}, {…}, {…}, {…}, {…}]
objectLink: "/namespaces/openshift-migration/clusters/host/namespaces/openshift-migration/restores/2d664340-1483-11eb-9b20-99dbdd85b490-mltbn"
__proto__: Object
```

```
kind: "PodVolumeRestore"
namespace: "openshift-migration"
name: "cb626470-0367-11eb-9229-7fe8551ebd2a-xcblt-pjtgq"
objectLink: "/namespaces/openshift-migration/clusters/host/namespaces/openshift-migration/podvolumerestores/cb626470-0367-11eb-9229-7fe8551ebd2a-xcblt-pjtgq"
__proto__: Object
```
```
kind: "Plan"
namespace: "openshift-migration"
name: "fffdsfa"
children: [{…}]
objectLink: "/namespaces/openshift-migration/plans/fffdsfa"
__proto__: Object
```
```
kind: "Migration"
namespace: "openshift-migration"
name: "2d664340-1483-11eb-9b20-99dbdd85b490"
children: (2) [{…}, {…}]
objectLink: "/namespaces/openshift-migration/migrations/2d664340-1483-11eb-9b20-99dbdd85b490"
__proto__: Object
```

```
kind: "Backup"
namespace: "openshift-migration"
name: "2d664340-1483-11eb-9b20-99dbdd85b490-bkbpn"
objectLink: "/namespaces/openshift-migration/clusters/src/namespaces/openshift-migration/backups/2d664340-1483-11eb-9b20-99dbdd85b490-bkbpn"
__proto__: Object
```

It looks like we will need to manipulate the data we receive from the discovery service and derive some ui specific state to indicate which cluster it should be run on. That way we can show this to the user in an elegant way. What are your thoughts @vconzola ... add more text to the dropdown selection to indicate "run on source" etc?

### PR DESCRIPTION
**Pull request by ibolton336**
Closes #1033

### CODE REVIEW

**Comment by eriknelson** [comment]
@ibolton336 looks like this guy is failing the linter, trying to test in the meantime

**Comment by eriknelson** [comment]
image

Saw this copying the backup

**Comment by eriknelson** [comment]
image
And copying the Restore

File: src/app/home/pages/PlanDebugPage/components/TreeActionsDropdown.tsx [hunk 1]
```diff
@@ -74,4 +74,13 @@ const TreeActionsDropdown: React.FunctionComponent<ITreeActionsDropdownProps> =
   );
 };
 
-export default TreeActionsDropdown;
+export default connect(
+  (state) => {
```
Comment (eriknelson) [inline src/app/home/pages/PlanDebugPage/components/TreeActionsDropdown.tsx:78]: Is this safe to remove if it's not returning anything?

**Review by eriknelson — CHANGES_REQUESTED** [review]
Small code q and linter updates

**Comment by eriknelson** [comment]
@vconzola Could you ack?

**Comment by vconzola** [comment]
@ibolton336 Minor thing and can be done in a different PR if you need to merge this right away @eriknelson , but the "Run the oc command..." text should under the alert title as shown in the second example under Inline variations here: https://www.patternfly.org/v4/components/alert

File: src/app/home/pages/PlanDebugPage/components/TreeActionsDropdown.tsx [hunk 2]
```diff
@@ -1,34 +1,42 @@
...
-    el.value = `oc get ${getFullKindName(rawNode.kind)} -n ${rawNode.namespace} ${rawNode.name}`;
+    const { ocCommand, clusterType } = getOCCommandAndClusterType(rawNode);
+    setClusterType(clusterType);
+    el.value = ocCommand;
     clipboard.appendChild(el);
     el.select();
     document.execCommand('copy');
     clipboard.removeChild(el);
+    copiedToClipboard(
+      `Copied to clipboard! Run the oc get command on
```
Comment (apinnick) [inline src/app/home/pages/PlanDebugPage/components/TreeActionsDropdown.tsx:35]: @eriknelson 
Suggested rewrite of error mesage: 
"Command copied to clipboard. Run 'oc get <resource>' on the ${clusterType} cluster to view resource details."

File: src/app/home/pages/PlanDebugPage/helpers.ts [hunk 3]
```diff
@@ -1,10 +1,64 @@
...
+        clusterType: 'target',
+      };
+    case 'PodVolumeRestore':
+      return {
+        ocCommand: `oc get ${getFullKindName(kind)} -n ${namespace} ${name}`,
+        clusterType: 'target',
+      };
+    default:
+      return {
+        ocCommand: `Failed to copy command!`,
```
Comment (apinnick) [inline src/app/home/pages/PlanDebugPage/helpers.ts:60]: @eriknelson 
Suggest you change 'Failed to copy command!' to "Command was not copied."

**Review by eriknelson — CHANGES_REQUESTED** [review]
Suggested message changes

**Review by eriknelson — APPROVED** [review]

**Pull request merged**